phantomgames/GameDetails/GameDetailViewController.swift

```diff
@@ -0,0 +1,19 @@
+//
+//  GameDetailsViewController.swift
+//  phantomgames
+//
+//  Created by Aphiwe Shozi on 2024/05/16.
+//
+
+import UIKit
+
+class GameDetailViewController: UIViewController {
```

Comment:
dele if not being used.

Reply:
this goes for all GameDetails page and Category files